astropy/utils/iers/iers.py

```diff
@@ -40,6 +40,7 @@
 # IERS-A default file name, URL, and ReadMe with content description
 IERS_A_FILE = 'finals2000A.all'
 IERS_A_URL = 'ftp://anonymous:mail%40astropy.org@gdc.cddis.eosdis.nasa.gov/pub/products/iers/finals2000A.all'  # noqa: E501
+IERS_A_URL_USNO = 'https://maia.usno.navy.mil/ser7/finals2000A.all'
```

Comment:
I am not sure about adding yet another config item. At this rate, might as well refactor to turn it into a list of mirrors... Also wasn't that FTP supposed to be "temporary", @eteq ?

Reply:
Since this is a dumpster fire right now, I am looking for a quick fix that does not change any API. We can talk about nice refactoring for v5.1 but this PR is going to be backported.

Do you think just swapping out the `IERS_A_URL` is good enough?

Reply:
update swaps out the IERS_A_URL and removes the extra bits I had added to the config